### PR TITLE
follow Docker official images recommendation

### PR DESCRIPTION
**Pull request by hilmarf**
## Description

Publishes latest changes on `main` branch as [package](https://github.com/open-component-model/ocm/pkgs/container/ocm).

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [x] 🤖 Build
- [ ] 🔁 CI
- [x] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

- Related Issue # (issue)
- Closes # (issue)
- Fixes # (issue)
> Remove if not applicable

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for tests # (issue/pr)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by morri-son — COMMENTED** [review]
macht Uwe das nicht hier https://github.com/open-component-model/ocm/tree/main/components/ocmcli mit OCM-Mitteln?

**Comment by morri-son** [comment]
Guter Punkt, wir sollten das latest tag beim release auch schreiben, oder?

**Comment by hilmarf** [comment]
> macht Uwe das nicht hier https://github.com/open-component-model/ocm/tree/main/components/ocmcli mit OCM-Mitteln?

yes and no... there is no `latest` tag available - and those images are created only, when there is a new release... so there is currently no 'nightly' build available as docker image

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+ARG GO_VERSION="1.22"
+ARG ALPINE_VERSION="3.19"
```
Comment (morri-son) [inline Dockerfile:2]:
```suggestion
ARG ALPINE_VERSION="3.19.1"
```

**Review by morri-son — APPROVED** [review]
Just saw that there's a patch for alpine 3.19 :)

File: Dockerfile (hunk 1 above)
Comment (hilmarf) [reply Dockerfile:2]: yes, I know... but there is no tag [alpine3.19.1](https://hub.docker.com/_/golang/tags?page=1&name=alpine3.19) for golang available

and if you check the image details [golang:1.22-alpine3.19](https://hub.docker.com/layers/library/golang/1.22-alpine3.19/images/sha256-5a99b4049412cd34ad6b4e0c9527ae6beb9ae82d787b4bf3f4eff7aa13fc577a?context=explore) it's using underneath **alpine:3, 3.19, 3.19.1, latest**

I think it's common practice in docker image tagging, when you leave out the `patch` part of the version, that `major.minor` should be the same as `major.minor`.*latest-available-patch*

**Pull request merged**